<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt 3D Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Viewer.qrc">
    <normaloff>:/resources/images/viewer_icon.png</normaloff>:/resources/images/viewer_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QQuickWidget" name="quickWidget">
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionConnect"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuScale_Mode">
     <property name="toolTip">
      <string>Set application scale mode (-scalemode [center,fit,fill] command line parameter)</string>
     </property>
     <property name="title">
      <string>Scale Mode</string>
     </property>
     <addaction name="actionCenter"/>
     <addaction name="actionScale_To_Fit"/>
     <addaction name="actionScale_To_Fill"/>
    </widget>
    <addaction name="actionBorder"/>
    <addaction name="menuScale_Mode"/>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionShowOnScreenStats"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionBorder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Matte</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionShowOnScreenStats">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Render Statistics</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
   <property name="toolTip">
    <string>Center content and use design dimensions</string>
   </property>
  </action>
  <action name="actionScale_To_Fit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale To Fit</string>
   </property>
   <property name="toolTip">
    <string>Fit content to available space but maintain aspect ratio</string>
   </property>
  </action>
  <action name="actionScale_To_Fill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale To Fill</string>
   </property>
   <property name="toolTip">
    <string>Use entire screen space for content, changing aspect ratio as necessary</string>
   </property>
  </action>
  <action name="actionToggle_Scale_Mode">
   <property name="text">
    <string>Toggle Scale Mode</string>
   </property>
   <property name="toolTip">
    <string>Toggle between the various scale modes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Viewer.qrc"/>
 </resources>
 <connections/>
</ui>
